package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "openaq-quality-checks",
+  "version": "1.0.0",
+  "description": "CLI for adding flags to OpenAQ data",
+  "main": "index.js",
+  "scripts": {
+    "main": "./index.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/openaq/openaq-quality-check.git"
+  },
+  "author": "Aimee Barciauskas",
+  "license": "ISC",
```

Comment:
I'm not sure how other openaq projects are licensed, so you might want to check that. Can likely be addressed in a future pr, too.

Reply:
good call, updated in https://github.com/openaq/openaq-quality-checks/pull/5/commits/582b60e5e7c6222862f73eb0dbf3f24d19ed46a1.